include/libtorrent/piece_block.hpp

```diff
@@ -41,14 +41,14 @@ namespace libtorrent
 	{
 		static const piece_block invalid;
 
-		piece_block() : piece_index(0) {}
+		piece_block() {}
```

Comment:
I suppose you still need this constructor here, since the default constructor won't be automatically generated if there is another constructor. but you can say ``= default`` instead of ``{}`` which may be nicer.